src/nginx/main/BUILD

```diff
@@ -218,6 +218,7 @@ pkg_tar(
         # The first occurrence of the duplicate file will be picked. In other
         # words, /etc/nginx/nginx.conf from ":nginx-conf-override" will
         # override that from "@nginx_pkgoss//:debian_overlay".
+        ":start_esp_tar",
```

Comment:
I don't want to pollute this folder.  It is hard to tell which targets are for production and which ones are for testing 

Can we move start_esp related things into 

start_esp/test/ folder

Reply:
Done.